linters/cmake-format/plugin.yaml

```diff
@@ -0,0 +1,30 @@
+version: 0.1
+tools:
+  definitions:
+    - name: cmake-format
+      runtime: python
+      package: cmake-format
+      shims: [cmake-format]
+      known_good_version: 0.6.13
+lint:
+  definitions:
+    - name: cmake-format
+      files: [cmake]
+      commands:
+        - name: lint
+          run: cmake-format --check ${target}
+          success_codes: [0, 1]
+          read_output_from: stderr
+          output: pass_fail
```

Comment:
What's the main value of having this lint subcommand here? Imo we can just include the `name: fix` subcommand and that should be sufficient. When I run `trunk check enable cmake-format; trunk check --force --filter=cmake-format linters/cmake-format`, I get `ERROR __main__.py:618: Check failed: linters/cmake-format/test_data/test2.in.cmake` as the lint output, so I'm not sure this is very actionable

Reply:
What are your thoughts on including `cmake-lint` here for the lint subcommand? We support turning on/off any number of linter subcommands, so we can start with some disabled by default.

Reply:
I see now, this was just me not fully understanding how this configuration worked. Only formatting should do.

Regarding `cmake-lint` I don't think it adds too much value as a tool. It mostly complains about lines being too long and bad indentation, which are corrected by the formatter.